pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -460,36 +460,36 @@ def sort_trigs(trial_dict, trigs, slide_dict, seg_dict):
     sorted_trigs = {}
 
     # Begin by sorting the triggers into each slide
-    # New seems pretty slow, so run it once and then use deepcopy
-    tmp_table = glsctables.New(glsctables.MultiInspiralTable)
     for slide_id in slide_dict:
-        sorted_trigs[slide_id] = copy.deepcopy(tmp_table)
-    for trig in trigs:
-        sorted_trigs[int(trig.time_slide_id)].append(trig)
+        sorted_trigs[slide_id] = []
+        for i in range(trigs['network/event_id'].len()):
+            slide_id = trigs['network/slide_id'][i]
+            sorted_trigs[slide_id].append(i)
```

Comment:
Using `slide_id` to run over `slide_dict` and assign `trigs['network/slide_id'][i]` makes these lines of code hard to parse.

```suggestion
        for i, _ in enumerate(trigs['network/event_id']):
            idx = trigs['network/slide_id'][i]
            sorted_trigs[idx].append(i)
```